### PR TITLE
[NMS] Add custom line chart to custom metrics

### PR DESCRIPTION
**Pull request by karthiksubraveti**
#1398  Summary
- Added custom linechart to custom metrics component
- This enables us to query from prometheus and also query events directly and show them on the same chart across similar intervals
- Cleaned up other associated components

## Test Plan
- yarn test succeeds
event_alert_gif

### CODE REVIEW

File: nms/app/packages/magmalte/app/components/EventAlertChart.js [hunk 1]
```diff
@@ -13,17 +13,33 @@
  * @flow strict-local
  * @format
  */
+<<<<<<< HEAD
```
Comment (Scott8440) [inline nms/app/packages/magmalte/app/components/EventAlertChart.js]: whoops

File: nms/app/packages/magmalte/app/components/EventAlertChart.js [hunk 2]
```diff
@@ -37,45 +53,175 @@ const CHART_COLORS = [
...
+
+    fetchAllData();
+  }, [start, end, enqueueSnackbar, networkId]);
+
+  if (isLoading) {
+    return <LoadingFiller />;
+  }
+
   return (
+<<<<<<< HEAD
```
Comment (Scott8440) [inline nms/app/packages/magmalte/app/components/EventAlertChart.js]: whoops

File: nms/app/packages/magmalte/app/components/EventAlertChart.js [hunk 3]
```diff
@@ -37,45 +53,175 @@ const CHART_COLORS = [
...
+    });
+  }
+
+  const alertPromResp = await MagmaV1API.getNetworksByNetworkIdPrometheusQueryRange(
+    {
+      networkId: networkId,
+      start: start.toISOString(),
+      end: end.toISOString(),
+      step: getStepString(delta, unit),
+      query: 'sum(ALERTS)',
```
Comment (Scott8440) [inline nms/app/packages/magmalte/app/components/EventAlertChart.js:110]: Have you tested that this gives you the number you expect? I haven't looked too much into this metric specifically and I don't know if there's redundancy or something in it where `sum` might give too high of a number. It's probably correct though.

**Review by Scott8440 — APPROVED** [review]
Looks good to me. Just a question about the actual query and fix the git stuff before landing

File: nms/app/packages/magmalte/app/components/EventAlertChart.js (hunk 1 above)
Comment (karthiksubraveti) [reply nms/app/packages/magmalte/app/components/EventAlertChart.js]: surprised, i was able to rebase continue with conflicts. I thought I had saved and added the file.

File: nms/app/packages/magmalte/app/components/EventAlertChart.js (hunk 3 above)
Comment (karthiksubraveti) [reply nms/app/packages/magmalte/app/components/EventAlertChart.js:110]: Unfortunately I haven't been able to verify alerts component yet. Currently it shows 2 alerts, but no alerts could be pulled through API. Once orc8r upgrades, I will verify and change it if necessary. Thanks for going through this.

**Review by andreilee — APPROVED** [review]

**Pull request merged**